examples/connect/kafka-connect-build.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: kafka.strimzi.io/v1beta2
```

Comment:
also this file it should not be here because it's leveraging a temporary registry (24h) where we push Kafka Connect builds. You should remove it, can you confirm @kyguy ? I remember we had this discussion with Jakub in the past as well.

Reply:
@ppatierno you are right [1]

[1] https://github.com/jboss-container-images/amqstreams-1-openshift-image/pull/382

Reply:
Removed